templates/java/build-alt.gradle.mustache

```diff
@@ -41,12 +51,19 @@ protobuf {
     grpc {
       artifact = 'io.grpc:protoc-gen-grpc-java:{{{dependencies.grpc.java.version}}}'
     }
+    gapic {
+      path = locateGapicPluginMethod()
+    }
   }
   generateProtoTasks {
     all()*.plugins {
       grpc {
         outputSubDir = 'java'
       }
+      gapic {
+        outputSubDir = 'java'
+        option '{{{api.gapicYaml}}}'
+      }
```

Comment:
The changes are:
- Add api-common dependency
- Add locateGapicPluginMethod that locates gapic_plugin.py (note that this requires gapic_plugin.py to be installed)
- Add the gapic{} blocks to the protobuf config. This configures the gradle protobuf plugin to invoke the gapic plugin, in the same way that the grpc{} blocks invoke the java grpc plugin. The path variable points to the plugin executable, in this case gapic_plugin.py, and the option variable is an argument that should be passed to the proto plugin via the protoc command line, in this case the path to the gapic yaml